gavel/controllers/admin.py

```diff
@@ -14,7 +14,7 @@
 
 ALLOWED_EXTENSIONS = set(['csv', 'xlsx', 'xls'])
 
-@app.route('/admin/')
+@app.route(settings.BASE_PATH + 'admin/')
```

Comment:
There must be a better way of handling this than using string concatenation of the base path in each of the route declarations. Perhaps `APPLICATION_ROOT` (see [here](http://flask.pocoo.org/docs/0.12/config/#builtin-configuration-values)) is the right way to do this.

Reply:
https://stackoverflow.com/a/36033627/1464290
> You should note that the APPLICATION_ROOT is NOT for this purpose.

Welp. A different method than string concatenation of the bath path seems more complicated. Unfortunately, I don't fully understand how to implement this feature a different way. While it may not be ideal, it definitely works.